.github/workflows/chain-interaction.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: 1.66.0
+          toolchain: nightly
```

Comment:
Change it to nightly since backtraces is the nightly feature. See:
https://github.com/CosmWasm/cosmwasm/blob/main/packages/std/Cargo.toml#L28